### DISCUSSION TITLE
Duplicate device after upgrade & a few other errors

### DISCUSSION THREAD

**Discussion by jaeti01** [Blackymas/NSPanel_HA_Blueprint#2266] [General]
Hello \*.\*,

After I upgraded my panel to BP 4.3.11 & ESPHome 2024.8.3, I got a bunch of errors and struggle how to fix it. 
First, the device now shows up twice:
![image](https://github.com/user-attachments/assets/07a24795-d098-416e-8da6-f3edd430154a)
I used to have MQTT enabled on the device, this could have caused that. But I have disabled it for the time being. 
It would anyway be great if that could be supported, as MQTT is my center of everything.

I also get the following error message:

```
The sensor "Current Page" (sensor.45654a6df8e3906f04ac37a95691d03e_current_page) is unavailable.

Panel: nspanel-buero (45654a6df8e3906f04ac37a95691d03e)
Blueprint version: 4.3.11
Sensor's expected entity_id: sensor.45654a6df8e3906f04ac37a95691d03e_current_page
```

The device shows all sensors twice:

![image](https://github.com/user-attachments/assets/336c1eaf-e90c-40e8-9187-a30e8695364d)

and  finally the panel itself shows a red HA icon in the upper right corner and does not respond to any touch. 
Any help is appreciated! TIA

**Comment by edwardtfn** [comment]
> and finally the panel itself shows a red HA icon in the upper right corner and does not respond to any touch.

This indicates an issue with the communication with the Blueprint, which could also be related to the duplicated entities.

I would suggest you remove the integration of this panel under Settings > Devices and Services > ESPHome, then restart Home Assistant and the panel should be detected again, recreating all the entities.
Please do no remove from the ESPHome dashboard.
There's no need to recreate the automation, but please make sure the Blueprint is the latest version.

More info here: https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/main/docs/error_initializing.md#blueprint-is-not-detected

**Reply by jaeti01** [reply]
Thank you. Finally got it working.  I went through deleting it  a couple of times with the error reoccurring until I re-installed ESPHome on  it, re-assigned the blueprint to the device. Not sure what finally fixed it, but it's running now. 
Thanks for your help!
